.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name": "wadm",
+	"image": "mcr.microsoft.com/devcontainers/base:bullseye",
```

Comment:
You can consider using rust image base instead of adding rust feature if just using some debian base: 
- <https://github.com/devcontainers/images/tree/main/src/rust>